_docker/docker-compose.yml.erb

```diff
@@ -92,6 +127,10 @@ awestruct:
     - DRUPAL_HOST_PORT=<%=ENV['DRUPAL_HOST_PORT']%>
     - DRUPAL_HOST_IP
     - HOST_TO_TEST
+    - SELENIUM_HOST
+    - DRIVER
+    - PARALLEL_TEST
+
```

Comment:
+    - SELENIUM_HOST
 +    - DRIVER
 +    - PARALLEL_TEST
I don't see why these are needed. awestruct is not responsible for testing